Makefile

```diff
@@ -0,0 +1,12 @@
+SHELL := /bin/bash # Use bash syntax
```

Comment:
Makefile instead of bash script?

Reply:
Makefile can call a bash script instead?

Reply:
I believe makefiles are just more developer friendly.
make <something> sounds better to me than running a bash script. (Just a personal opinion.)

In future there can be more make targets and IMO these 3-4 commands are not worthy of separate bash script.